src/BloomBrowserUI/bookLayout/basePage.less

```diff
@@ -286,9 +286,14 @@ H2 {
 	.bloom-imageContainer {
 		width: 99%;
 		height: 98%;
-		/*above so buttons show*/
+		// above so buttons show
 		z-index: 1000;
+		// Solves BL-1033 small picture frames cycling red overflow
+        overflow: hidden;
 	}
+    .bloom-editable {
+        overflow-y: hidden;
```

Comment:
Won't this prevent real overflowing text from being visible below the bottom of the box? I thought JohnH recently changed things on purpose to allow that to happen.